Top-Level Requirements.md

```diff
@@ -2,22 +2,23 @@
 ## List of Terms
 * East **shall** be defined as the direction perpendicular and away to the face of the designated mining receptacle.
 ## Requirements
-### 1.	Executive System
-* 1.1 The executive system **shall** process mission control commands.  
-* 1.2 The executive system **shall** start and manage autonomous operation.  
-* 1.3 The executive system **shall** monitor and delegate system responsibilities and state in between mission subsystems.  
-* 1.4 The executive system **shall** support two classes of commands:  
-    * 1.4.1 The Autonomous Command: Commences the main mission phases and tracks autonomous mission state.  
-    * 1.4.2 Teleop Commands: A lightweight command processing mode which contains user commands to be fed to control systems.  
-* 1.5 The executive system **shall** track mission progress and mission time elapsed.  
-* 1.6 The executive system **shall** be able to preempt all subsystem.  
-* 1.7 The executive system **shall** return the system to a safe state upon failure.  
+### 1.  Executive System
+* 1.1 The executive system **shall** process mission control commands.
+* 1.3 The executive system **shall** monitor and delegate system responsibilities and state in between mission subsystems.
```

Comment:
What happened to 1.2?

Reply:
Thanks for catching that, markdown, should add an ordered list feature in future iterations of the language.